### DISCUSSION TITLE
How to Append SVG to a Div with ID=HelloWorld ?

### DISCUSSION THREAD

**Discussion by MichaelCropper** [d3/d3#3653] [Q&A]
Current working example I have is with this line of code;

```
document.body.appendChild(svg);
```

I've still not found any documentation that explains what the above line of code is actually doing. I'm assuming this is some D3 stuff in the background and/or some other poorly more recent JavaScript stuff that I'm just not that familiar with. 

What is the syntax / working example to bind D3 to something like;

```
<div id="HelloWorldDiv"></div>
```

i.e. a basic HelloWorld version of something like this;
```
document.getElementById("HelloWorldDiv").appendChild(svg)
```

There are **way** too many assumptions being made in the docs/examples online I've managed to find. Struggling at every baby step at the minute. 

I've actually attempted the above already and D3 seems to be **trying** to do something, but it's not working properly. Seems as though it's creating the two Bubbles, but like Nested Bubbles (imagine a Russian Doll or an Onion with the layers) and it's only showing 1/4 of the visualisation. Seems strange.

**Comment by Fil** [comment]
This is not a question about D3, you will need to find some other resource to learn about JavaScript and the DOM.
